src/shared/vpick.c

```diff
@@ -85,8 +85,13 @@ static int format_fname(
                         return -ENOMEM;
         }
 
-        if (filter->suffix && !strextend(&fn, filter->suffix))
-                return -ENOMEM;
+        if (!strv_isempty(filter->suffix)) {
+                if (strv_length(filter->suffix) > 1)
```

Comment:
Hmm, I prefer to move this check to the beginning of the function? I.e. give up early if we can

Reply:
fn is used next so it has to be here, and it seems nicer to use the suffix in the same place as it is checked

Reply:
(I only meant the `strv_length` check, if we place this at the beginning, pointless string concatenation could be avoided and it's more obvious IMO)

-> #32370